src/option_parser.c

```diff
@@ -249,25 +284,10 @@ int load_ini_file(FILE * fp)
                 }
 
                 *equal = '\0';
-                char *key = g_strstrip(start);
-                char *value = g_strstrip(equal + 1);
+                char *key = g_strdup(g_strstrip(start));
```

Comment:
Why call `g_strdup` here? It's better to leave it to `add_entry` to handle memory. (Even though it looks like `value` was `strdup`-ed here earlier, this is as good a time as any to change that)